designer/client/src/components/toolbars/scenarioDetails/ScenarioLabels.tsx

```diff
@@ -340,6 +340,7 @@ export const ScenarioLabels = ({ readOnly }: Props) => {
                                         variant="outlined"
                                         disabled={readOnly}
                                         label={option.title}
+                                        sx={{ borderRadius: "16px", m: "2px" }}
```

Comment:
As we discussed, you can use [styled component](https://mui.com/system/styled/) here. It's a non-blocking comment, you can resolve it.

Reply:
Fixed (by using styled component :) )